.gitlab/build_quartz.yml

```diff
@@ -43,6 +43,7 @@ release_resources_build_quartz:
 .main_build_with_deps_on_quartz:
   stage: q_build_with_deps
   extends: [.srun_build_with_deps_script, .on_quartz, .main_workflow]
+  needs: []
```

Comment:
What did this do?

Reply:
It allows the full Quartz build to run in parallel with the Lassen build (which was already the case for the partial PR builds, I just missed it).

Reply:
such a clear syntax.

Reply:
@white238 that’s sort of a tweak.
What it really says is: "This jobs does not need any other job to start" allowing the job to start even if it is in a later stage.
I general, it’s important to keep in mind that once a `needs` keyword is added to a job, it is not restricted to the synchronization barriers between stages anymore.